### DISCUSSION TITLE
翻译折叠功能的应用场景探索

### DISCUSSION THREAD

**Discussion by YiiGuxing** [YiiGuxing/TranslationPlugin#920] [Ideas]
翻译且折叠：
![image](https://user-images.githubusercontent.com/31927848/147306513-67566d6b-f288-4b75-9018-56c23be3c298.png)

**Comment by vission8899** [comment]
能不能全项目中 所有在单词本数据库里的单词  都自动翻译  

最好形式是【鼠标放上去自动展示翻译结果 项目中保留原文】

这样可以增加对单词的印象与记忆 不然单词本的功能比较孱弱 基本上放进单词本的单词都吃灰了 下次碰到也不知道是个啥
